docs/BrandColors.mdx

```diff
@@ -5,7 +5,7 @@ import LinkTo from '@storybook/addon-links/react';
 
 # Brand Colors (first tier)
 
-Brand specific colors used in the first tier of design token abstraction. They **should not** be used in the code directly but referenced by [**Theme Colors**](/docs/colors-themecolors--light-theme-colors) used in second tier tokens.
+Brand specific colors used in the first tier of design token abstraction. They **should not** be used in the code directly but referenced by [**Theme Colors**](/docs/colors-themecolors--light-theme-colors) used in second tier tokens. Most brand color progressions were generated using [0to255](https://0to255.com/037dd6)
```

Comment:
Adding reference to tool that original color progressions were used to make. Not apart of this PR but aligns with the design files and I think that it helps gives background to how they were chosen.